iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift

```diff
@@ -10,15 +10,16 @@ import Foundation
 import StreamChat
 import UIKit
 
-/// A cell to display a high-level view of a conversation's message. Displays a limited number of recent replies to the message.
-/// The user's replies and other replies are put in two stacks (along the z-axis), with the most recent reply at the front (visually obscuring the others).
-class ConversationMessageCell: UICollectionViewCell, ConversationMessageCellLayoutDelegate {
+/// A cell to display a high-level view of a conversation's message. Displays a limited number of recent messages in a conversation.
+/// The user's messages and other messages are put in two stacks (along the z-axis),
+/// with the most recent message at the front (visually obscuring the others).
+class ConversationMessagesCell: UICollectionViewCell, ConversationMessageCellLayoutDelegate {
 
     // Interaction handling
-    var handleTappedMessage: ((Messageable) -> Void)?
-    var handleDeleteMessage: ((Messageable) -> Void)?
+    var handleTappedConversation: ((Messageable) -> Void)?
+    var handleDeleteConversation: ((Messageable) -> Void)?
```

Comment:
This cell now represents a whole conversation so you're really tapping on the whole conversation. Later on we can get more granular for opening up threads.